tests/conftest.py

```diff
@@ -201,3 +204,43 @@ def compare(received_raw: bytes, golden_name: str):
         assert diff_pixels == 0
 
     return compare
+
+
+class RemoteServer:
+    def __init__(self, browser_name: str) -> None:
+        options = {}
+        if is_win():
```

Comment:
Did you mean not `is_win` here as `os.setsid` is not available on windows

Reply:
oh yeah. I'm thinking about using node there directly, that would ease-up our life with killing the whole sub-tree of processes. See here how Java is doing it:

https://github.com/microsoft/playwright-java/blob/20322470b5f23698faf7b19704015c822861e360/playwright/src/test/java/com/microsoft/playwright/TestBrowserTypeConnect.java#L57-L58

Reply:
Yeah it would be better